roles/lib_utils/filter_plugins/openshift_master.py

```diff
@@ -482,9 +482,7 @@ def translate_idps(idps, api_version):
 
     @staticmethod
     def certificates_to_synchronize(hostvars, include_keys=True, include_ca=True):
-        ''' Return certificates to synchronize based on facts. '''
-        if not issubclass(type(hostvars), dict):
-            raise errors.AnsibleFilterError("|failed expects hostvars is a dict")
+        ''' Return certificates to synchronize. '''
```

Comment:
@jctanner  
Remove hostvars, and for calling this filter, use
{{ True | certificates_to_synchronize }}

and {{ False | ... }}

Be sure to trim out the include_keys portion for the filter args in the second usage that @mtnbikenc linked.